tests/cli/test_cli.py

```diff
@@ -1354,6 +1387,56 @@ def test_rm(ctx):
     _run_command([cl, 'rm', uuid])
     check_equals(disk_used, _run_command([cl, 'uinfo', 'codalab', '-f', 'disk_used']))
 
+    # Test with running bundle
+    disk_used = _run_command([cl, 'uinfo', 'codalab', '-f', 'disk_used'])
+    uuid = _run_command([cl, 'run', 'head -c 50 /dev/zero > test.txt'])
+    wait_until_state(uuid, State.READY)
+    data_size = _run_command([cl, 'info', uuid, '-f', 'data_size'])
+    check_equals(
+        str(int(data_size) + int(disk_used)),
+        _run_command([cl, 'uinfo', 'codalab', '-f', 'disk_used']),
+    )
+    _run_command([cl, 'rm', uuid])
+    check_equals(disk_used, _run_command([cl, 'uinfo', 'codalab', '-f', 'disk_used']))
+
```

Comment:
Do these test code paths end up calling `cleanup_existing_contents`?

Reply:
Ah, shoot. No, I don't think so. Let me add one more test for that before I merge.

Reply:
Thanks for catching that!